index.html

```diff
@@ -411,5 +412,6 @@ <h2>Contact me</h2>
         </div>
```

Comment:
- Optional.
Your contact-me section is also breaking. Consider fixing it.
![image](https://user-images.githubusercontent.com/33268927/173549920-dcbee858-cab2-433f-ab49-4a6f2bbe0744.png)